src/engraving/rendering/score/tdraw.cpp

```diff
@@ -2189,7 +2189,7 @@ void TDraw::draw(const Note* item, Painter* painter)
 
     bool negativeFret = item->negativeFretUsed() && item->staff()->isTabStaff(item->tick());
 
-    Color c(negativeFret ? config->criticalColor() : item->curColor());
+    Color c(negativeFret && !item->deadNote() ? config->criticalColor() : item->curColor());
     painter->setPen(c);
```

Comment:
I think this Color definition and painter->setPen command can be removed completely, cause `c` is only used in one other place (where it should just be replaced with `item->curColor()`). It also looks wrong to me in any case, cause we should only ever use the warning color when drawing on the editor, not when drawing in print, so setting the pen to `criticalColor` should be always have a `!score->printing()` condition. Plus, it seems to me that in case of negative frets we already draw a red box in the background so it doesn't make sense for the text to also be red.